mapbox/services/static_style.py

```diff
@@ -91,6 +92,15 @@ def image(self, username, style_id, lon=None, lat=None, zoom=None, features=None
         if before_layer is not None:
             params['before_layer'] = before_layer
 
+        # twox as a deprecated alias for retina
+        if retina is None:
+            if twox is not None:
+                warnings.warn('twox is a deprecated alias for retina', DeprecationWarning)
+                retina = twox
+        else:
+            if twox is not None:
+                raise errors.ValidationError('Conflicting args; Remove twox and use retina')
+
```

Comment:
@perrygeo I've been persuaded by Brett Slatkin (in "Effective Python") that we should draw a line between Python deprecation warnings, which are for core Python developers, and our own deprecation warnings, which are for our SDK users. I'm inclined to switch to a `MapboxDeprecationWarning` that derives from `UserWarning` and isn't ignored by default. What would you think of that?